addons/website/data/website_data.xml

```diff
@@ -173,7 +173,7 @@
                                         </div>
                                         <h5 class="text-center">
                                             <span class="fa fa-check-circle"/>
-                                            <span>Your message has been sent <b>successfully</b></span>
+                                            <span>Your message has been sent <span style="font-weight: bold;">successfully</span></span>
```

Comment:
Commit message:
- I would say [REF] instead of [FIX] for this one
- The first and the last sentences should be reviewed, probably not end the first one with a point but with a comma for instance?